i18n/locales/source/course_content/2017/csd2-2017.json

```diff
@@ -229,7 +229,7 @@
   "https://studio.code.org/s/csd2-2017/stage/8/puzzle/1": {
     "dsls": {
       "content1": "description here",
-      "markdown": "# Overview\n\nIn this lesson covers common issues that arise when designing web pages in HTML. The class will correct errors in a sequence of increasingly complex web pages found on Code Studio and learn the importance of comments, whitespace, and indentation as tools for making web pages easier to read.\n## Vocabulary\n\n* **Bug** - Part of a program that does not work correctly.\n* **Comment** - A comment is a programmer-readable note in the source code of a computer program.\n* **Debugging** - Finding and fixing problems in your algorithm or program.\n* **Indentation** - The placement of text farther to the right, or left, to separate it from surrounding text which helps to convey the program's structure.\n* **Whitespace** - Whitespace refers to any character that shows up as a blank space on the screen, such as a space, a tab, or a new line. Whitespace helps separate different parts of the document to make it easier to read.\n\n## Introduced Code\n\n*  **[Comment](//curriculum.code.org/documentation/weblab/comment/)** - `<!-- -->`\n"
+      "markdown": "# Overview\n\nIn this lesson covers common issues that arise when designing web pages in HTML. The class will correct errors in a sequence of increasingly complex web pages found on Code Studio and learn the importance of comments, whitespace, and indentation as tools for making web pages easier to read.\n## Vocabulary\n\n* **Bug** - Part of a program that does not work correctly.\n* **Comment** - A comment is a programmer-readable note in the source code of a computer program.\n* **Debugging** - Finding and fixing problems in your algorithm or program.\n* **Indentation** - The placement of text farther to the right, or left, to separate it from surrounding text which helps to convey the program's structure.\n* **Whitespace** - Whitespace refers to any character that shows up as a blank space on the screen, such as a space, a tab, or a new line. Whitespace helps separate different parts of the document to make it easier to read.\n\n## Introduced Code\n\n*  **[Comment](//studio.code.org/docs/weblab/comment/)** - `<!-- -->`\n"
```

Comment:
similarly, this content is generated from our level definitions; this change would normally be overwritten by the next sync, but you also made the same change to the corresponding level file so that should be fine.